editor/editor_inspector.h

```diff
@@ -511,6 +513,13 @@ class EditorInspector : public ScrollContainer {
 
 	void _update_inspector_bg();
 
+	ConfirmationDialog *add_meta_dialog;
+	LineEdit *add_meta_name;
+	OptionButton *add_meta_type;
```

Comment:
```suggestion
	ConfirmationDialog *add_meta_dialog = nullptr;
	LineEdit *add_meta_name = nullptr;
	OptionButton *add_meta_type = nullptr;
```
This should fix the crash.

Reply:
Thanks! Not doing much code recently is making me rusty.